### PR TITLE
fix: support trailing slash

### PR DESCRIPTION
**Pull request by RafaelGSS**
Refs: https://openjs-foundation.slack.com/archives/C019Y2T6STH/p1720463991463229

### CODE REVIEW

**Comment by RafaelGSS** [comment]
Not sure why lint is complaining... If I remove the `\/?` from _Fixes_, tests won't pass.

**Comment by Trott** [comment]
> Not sure why lint is complaining... If I remove the `\/?` from _Fixes_, tests won't pass.

It's complaining because you're backslash-escaping it in a string. You don't need to backslash-escape a slash character in a string, only in a regex literal.

File: lib/rules/fixes-url.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const id = 'fixes-url'
 const github = new RegExp('^https://github\\.com/[\\w-]+/[\\w-]+/' +
-  '(issues|pull)/\\d+(#issuecomment-\\d+|#discussion_r\\d+)?$'
+  '(issues|pull)/\\d+(#issuecomment-\\d+|#discussion_r\\d+)?\/?$'
```
Comment (Trott) [inline lib/rules/fixes-url.js:3]:
```suggestion
  '(issues|pull)/\\d+(#issuecomment-\\d+|#discussion_r\\d+)?/?$'
```

**Review by Trott — APPROVED** [review]
Looks good to me. I assume that since you're a releaser, you've taken into consideration whether this might break relevant tools like the changelog maker. (I don't imagine it would break those tools. I'm just mentioning it in case you or someone else who deals with that tooling more often might think otherwise or want to give it a bit more consideration/testing.)

**Comment by RafaelGSS** [comment]
> I assume that since you're a releaser, you've taken into consideration whether this might break relevant tools like the changelog maker. (I don't imagine it would break those tools. I'm just mentioning it in case you or someone else who deals with that tooling more often might think otherwise or want to give it a bit more consideration/testing.)

Considering we are not dropping support but adding trailing slash support, it should not impact the existing release workflow or tools like changelog maker. But, I'll monitor it over the week.

**Review by richardlau — APPROVED** [review]

**Comment by richardlau** [comment]
I think the one possible change in behavior might be branch-diff _if_ the PR-URL in the commits being compared differed only in whether they ended in a slash or not (since they'd no longer match).

**Comment by RafaelGSS** [comment]
> I think the one possible change in behavior might be branch-diff _if_ the PR-URL in the commits being compared differed only in whether they ended in a slash or not (since they'd no longer match).

That's correct. Let me see this first.

**Comment by RafaelGSS** [comment]
Created https://github.com/nodejs/branch-diff/pull/70

**Pull request merged**